Lib/socket.py

```diff
@@ -28,6 +28,8 @@
 socket.setdefaulttimeout() -- set the default timeout value
 create_connection() -- connects to an address, with an optional timeout and
                        optional source address.
+create_server() -- creates a TCP server socket and binds it to a specified
```

Comment:
I would just say `TCP socket`, am no English native speaker thoough.

Reply:
Explicit is better than implicit - [PEP 20](https://www.python.org/dev/peps/pep-0020/) :)

Reply:
I applied Joannah's suggestion.